### DISCUSSION TITLE
Don't know how to use LSP

### DISCUSSION THREAD

**Discussion by notramo** [helix-editor/helix#7370] [Ideas]
I see that there is a little blue bar in front of the lines that has LSP errors. But when I navigate to the line, nothing happens, and I don't know what to do to view the LSP message.
Ideas:

1. Suggest keybinding at the bottom of the screen.
2. Display LSP message at the bottom of the screen.
3. Anything except current behavior. I am too lazy to dig through documentation especially when I'm don't know what part should I look at.

**Comment by kirawi** [comment]
The blue line you're talking about is the diff gutter. LSP-related stuff would be a dot to the left of the line numbers (in the default config). You can also enable LSP messages in your config: https://docs.helix-editor.com/configuration.html#editorlsp-section
